system-test/storage.ts

```diff
@@ -1038,8 +1038,12 @@ describe('storage', () => {
         await setUniformBucketLevelAccess(bucket, true);
         await setUniformBucketLevelAccess(bucket, false);
 
-        const [aclAfter] = await file.acl.get();
-        assert.deepStrictEqual(aclAfter, aclBefore);
+        // Introduce a slight delay as it appears GCS might need time to
+        // propagate changes to the file.
+        process.nextTick(async () => {
```

Comment:
What if we did a loop instead - the `nextTick` may be fragile for backend changes or local async changes.

Feature Request for the backend (if it doesn’t exist): ACL push/pubsub notifications

Reply:
I've reworked this to check periodically (every 5s) to see if the call succeeds.